9/problem9.rb

```diff
@@ -0,0 +1,66 @@
+require 'date'
+require 'time_difference'
+
+puts "Enter your Birth date:(DD/MM/YYYY)"
+
+b_date=gets.chomp!
+
+splitted_b_array = b_date.split("/")
+
+bday = Integer(splitted_b_array[0])
+
+bmonth = Integer(splitted_b_array[1])
+
+byear = Integer(splitted_b_array[2])
+
+
+currentDate=DateTime.now
+createdDate=Date.new(byear,bmonth,bday)
+
```

Comment:
createdDate = Date.strptime(dob, '%d/%m/%Y')
you could have also use this to convert input string to required date format instead of doing this in four lines (lines# 10, 12, 14, 18)